duckduckgo.safariextension/html/global.html

```diff
@@ -219,7 +248,15 @@
             }
         }, false);
     }
-
+    
+    safari.application.addEventListener('navigate', function(evt) {
+        if(evt.target && evt.target.url){
+            var urlMatch = evt.target.url.match(/https?:\/\/duckduckgo.com\/\?q=.*/);
+            if(urlMatch && urlMatch[0]){
+                update_set_atb();
+            }
+        }
+    });
```

Comment:
Maybe we should avoid adding the listener entirely if `localStorage['atb'] and localStorage['atb_set']` don't exist? Otherwise this will run on every web request for every user.